numpy/core/src/umath/loops_arithmetic.dispatch.c.src

```diff
@@ -0,0 +1,132 @@
+/*@targets
+ ** $maxopt baseline
+ ** sse2 sse41 avx2 avx512_skx
+ ** vsx2
+ ** neon
+ **/
+#define _UMATHMODULE
+#define _MULTIARRAYMODULE
+#define NPY_NO_DEPRECATED_API NPY_API_VERSION
+
+#include "simd/simd.h"
+#include "loops_utils.h"
+#include "loops.h"
+#include "lowlevel_strided_loops.h"
+#include<signal.h> 
+// Provides the various *_LOOP macros
+#include "fast_loop_macros.h"
+
+//###############################################################################
+//## Unsigned Integers
+//###############################################################################
+/********************************************************************************
+ ** Defining the SIMD kernels
+ ********************************************************************************/
+#ifdef NPY_SIMD
+/**begin repeat
+ *  #sfx = u8, u16, u32, u64#
+ *  #type = npy_ubyte, npy_ushort, npy_uint, npy_ulong#
+ */
+
+static void simd_divide_by_scalar_contig_contig_@sfx@
+(const char *src, const @type@ scalar, char *dst,
+ int len)
+{
+    npyv_lanetype_@sfx@ *npyv_src = (npyv_lanetype_@sfx@ *) src;
```

Comment:
```suggestion
(const char *_src, const @type@ scalar, char *dst,
 int len)
{
    npyv_lanetype_@sfx@ *src = (npyv_lanetype_@sfx@ *) _src;
```
prefix `npyv_` is reserved for the SIMD interface, you can just add `_` to parm names instead.